js/views/discoveryScreen.js

```diff
@@ -50,7 +50,7 @@ var DiscoveryScreen = React.createClass({
     return (
       <div className="discoveryScreen" onMouseDown={this.handleMouseDown}>
         <div className={promoStyle}></div>
-        <DiscoveryPanel {...this.props} />
+        <DiscoveryPanel {...this.props} videosPerPage={6}/>
```

Comment:
Yes, it can be set in skin.json. Videos per page also needs to be responsive, according to mocks, breakpoint values are -
* sm - 2
* md - 6
* lg - 10